changelog.d/365.feature

```diff
@@ -0,0 +1 @@
+Use MediaProxy to serve authenticated Matrix media.
```

Comment:
Could we describe this in a little more detail? Does the admin need to perform any actions here?